Docs/landing/data/releases.toml

```diff
@@ -1,7 +1,7 @@
 current = "2.9.3"
```

Comment:
Probably this one should be 2.10.0?

Reply:
It will be changed during the release process